examples/io/elekta_average.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
@Eric89GXL to edit/define an averaging category in DACQ, you have to activate it first. Thus, the categories marked active probably are of interest to the user and the rest are not. Elekta's own averager ignores any non-active categories.

In your example, it looks like the averaging settings were not defined at all. I will beautify the repr for such cases.